css/employees.css

```diff
@@ -0,0 +1,375 @@
+
```

Comment:
i have noticed some duplicate styles like `body{
    width:100%;
    height:100%;
    font-family: "Poppins";
    color:#4b4d5a;
    overflow-x: hidden;
    display: flex;
}` in employees.css, role-details, roles.css.
To encourage the reusability of styles across the pages, move those shared styles to separate css file